models/Book.js

```diff
@@ -0,0 +1,44 @@
+let mongoose = require('mongoose');
+
+let BookSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    price: {
+        type: Number,
+        required: true
+    },
+    publishYear: {
+        type: Number,
+        required: true
+    },
+    description: {
+        type: String,
+        required: true
+    },
+    currentHealth: {
```

Comment:
Książki chyba jednak nie mają hp. Chyba że to ukryte quantity ;>

Reply:
Wywalone